pages/index.js

```diff
@@ -1,69 +1,13 @@
 import Head from 'next/head'
-import Image from 'next/image'
 import styles from '../styles/Home.module.css'
+import Header from '../templates/Header'
 
 export default function Home() {
   return (
     <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
+      <Head> 
+        <Header/>
```

Comment:
I don't think the Header component you defined should be rendered under the Head component and typically you would have this defined in a layout component since it would remain the same across certain pages